verification/curator-service/api/src/controllers/uploads.ts

```diff
@@ -62,7 +62,14 @@ export default class UploadsController {
         try {
             const uploads = await Source.aggregate([
                 { $unwind: '$uploads' },
-                { $replaceRoot: { newRoot: '$uploads' } },
+                {
+                    $project: {
+                        _id: 0,
```

Comment:
nit, but slightly clearer for this to be `false` IMO

Reply:
I agree, done.